src/main/java/core/basesyntax/data/ReportGenerator.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportGenerator {
+    public List<String> generateReport(Map<String, Integer> inventory) {
```

Comment:
Consider adding null checks for the `inventory` parameter to prevent potential `NullPointerException` if a null map is passed to the method.